gradio/components/chatbot.py

```diff
@@ -62,7 +62,7 @@ class Chatbot(Component):
     Also supports audio/video/image files, which are displayed in the Chatbot, and other kinds of files which are displayed as links. This
     component is usually used as an output component.
 
-    Demos: chatbot_simple, chatbot_multimodal
+    Demos: chatbot_simple, chatbot_multimodal, chatbot_core_components
```

Comment:
we don't currently support demos with any requirements on the docs, I can add support for it soon. But remove this for now as its erroring out on lite.